src/cpp/src/SIO/SIOTrackHandler.cc

```diff
@@ -50,13 +50,20 @@ namespace SIO {
     SIO_DATA( device ,  &(trk->_ndf)  , 1 ) ;
     SIO_DATA( device ,  &(trk->_dEdx) , 1  ) ;
     SIO_DATA( device ,  &(trk->_dEdxError) , 1  ) ;
+    SIO_DATA( device ,  &(trk->_nholes) , 1  ) ;
```

Comment:
This will need a version heck to keep backward compatibility, something like
```suggestion
    if (vers >= SIO_VERSION_ENCODE(2, 21) {
      SIO_DATA( device ,  &(trk->_nholes) , 1  ) ;
    }
```